tests/e2e/cucumber/steps/ui/adminSettings.ts

```diff
@@ -300,3 +300,33 @@ When(
     }
   }
 )
+
+When(
+  /^"([^"]*)" deletes the following (user|users) using the (batch actions|context menu)$/,
+  async function (
+    this: World,
+    stepUser: string,
+    _: string,
+    way: string,
```

Comment:
```suggestion
    actionType: string,
```
how about using this?